decidim-participatory_processes/app/cells/decidim/participatory_processes/process_group_g_cell.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
+    # This cell renders the Grid (:g) process card
+    # for an given instance of a Process
+    class ProcessGroupGCell < Decidim::CardGCell
+      private
+
+      def resource_path
+        Decidim::ParticipatoryProcesses::Engine.routes.url_helpers.participatory_process_group_path(model)
+      end
+
+      def resource_image_path
```

Comment:
This is also present in AssemblyGCell. ProcessGCell and ConferenceGCell

Reply:
Yes, all these models have an attachment with the same name. Since they are very direct methods I did not consider making a concern for it or move the method to the parent class (in this case the method would not be valid for all the child classes), but if you prefer it I can do it.

Reply:
it is ok like this. Thanks!